avocado/utils/astring.py

```diff
@@ -219,7 +219,7 @@ def string_safe_encode(string):
     try:
         return string.encode("utf-8")
     except UnicodeDecodeError:
-        return string.decode("utf-8").encode("utf-8")
+        return string.decode("utf-8", "ignore").encode("utf-8")
```

Comment:
Oops, I'm really blind.  There are actually encode and decode operations on the same line.  Never mind the previous comment.